### PR TITLE
Add targeted e2e tests

### PR DESCRIPTION
**Pull request by CodeByEddy**
Let people trigger e2e tests against a specific API URL

### CODE REVIEW

File: .github/workflows/targeted-e2e.yml [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+        uses: Xotabu4/selenoid-github-action@v2
+        with:
+          selenoid-start-arguments: |
+            --args "-timeout 300s" --browsers 'chrome:95.0;chrome:96.0;chrome:94.0'
+        
+
+      - name: Set Configuration - Staging
+        run:  |
+          echo 'UI_URL=https://staging.mangata.finance/' >> $GITHUB_ENV
+          echo 'API_URL={{ github.event.client_payload.api_url }}' >> $GITHUB_ENV
```
[the remark this answers is not in the document]
Comment (goncer) [reply .github/workflows/targeted-e2e.yml:63]: BTW, I added un the TestUI workflow the possibility to use v4.

**Review by goncer — APPROVED** [review]

File: .github/workflows/targeted-e2e.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (CodeByEddy) [reply .github/workflows/targeted-e2e.yml:63]: Did you happen to find a solution to this @goncer? :)

[the remark this answers is not in the document]
Comment (goncer) [reply .github/workflows/targeted-e2e.yml:63]: image
In e2e actions, you can find `Environment tests`
That can be used to run tests targeting an environment. Originally this workflow was for Ui tests but can target any specific script.

[the remark this answers is not in the document]
Comment (goncer) [reply .github/workflows/targeted-e2e.yml:63]: @eddyjlsh , @CodeByEddy  does this convince you? do you have any other ideas?

[the remark this answers is not in the document]
Comment (CodeByEddy) [reply .github/workflows/targeted-e2e.yml:63]: That looks good!

**Comment by CodeByEddy** [comment]
Re-running CI tests then merging

**Pull request closed without merging**